src/nanoemoji/glue_together.py

```diff
@@ -84,28 +95,105 @@ def _copy_svg(target: ttLib.TTFont, donor: ttLib.TTFont):
     target["SVG "] = donor["SVG "]
 
 
+def _cbdt_data_and_sizes(ttfont: ttLib.TTFont) -> Mapping[str, CbdtGlyphInfo]:
+    data = {}
+    for strike_data in ttfont["CBDT"].strikeData:
+        data.update(strike_data)
+
+    sizes = {}
+    for strike in ttfont["CBLC"].strikes:
+        for sub_table in strike.indexSubTables:
+            for name, (start, end) in zip(sub_table.names, sub_table.locations):
+                sizes[name] = end - start
+
+    assert data.keys() == sizes.keys(), f"{data.keys()} != {sizes.keys()}"
+
+    return {
+        glyph_name: CbdtGlyphInfo(data[glyph_name], sizes[glyph_name])
+        for glyph_name in data
+    }
+
+
+def _copy_cbdt(target: ttLib.TTFont, donor: ttLib.TTFont):
+    cbdt_glyph_info = _cbdt_data_and_sizes(donor)
+
+    # reorder the bitmap table to match the targets glyph order
+    # we only support square bitmaps so the strikes are all the same
+    # other than glyph names so we can just construct a new
+    # order that matches that of target
+    donor_order = list(cbdt_glyph_info.keys())
+    # confirm our core assumption that we successfully held glyph names stable
+    if not set(donor_order) <= set(target.getGlyphOrder()):
+        raise ValueError("Donor glyph names do not exist in target")
+    new_order = sorted(donor_order, key=target.getGlyphID)
+
+    # now we know the desired order, reshard into runs
+    # TODO duplicative of make_cbdt_table
+    strike_template = copy.deepcopy(donor["CBLC"].strikes[0])
+    clbc_index_template = copy.deepcopy(strike_template.indexSubTables[0])
+    strike_template.indexSubTables = []
+
+    cblc = donor["CBLC"]
+    cblc.strikes = []
```

Comment:
why is this modifying the donor instead of copying?

Reply:
I copy the first strike from donor to use as a template, then rebuild a new set of strikes (see clbc.strikes.append a few lines down) based on the reordered glyphs. I'll update the comment.

Reply:
...and indeed the copy template is pointless since I then free all the references. Removing.

Reply:
you removed the copies so the transplant kills the donor and organs can't be reused for a different target, I suppose it's fine since the method is private and only used inside main. might be worth mentioning that in a comment maybe.

Reply:
Rename donor to victim? :D

In this context I think it's fine, donors sole purpose is to become somewhat one with target ;)